lib/matplotlib/axes/_subplots.py

```diff
@@ -49,12 +47,13 @@ def __reduce__(self):
                 (axes_class,),
                 self.__getstate__())
 
+    @cbook.deprecated("3.4", alternative="get_subplotspec")
```

Comment:
The change in the return type needs to be flagged.

Reply:
sure